RevenueCatUI/Resources/es_419.lproj/Localizable.strings

```diff
@@ -1,4 +1,4 @@
-"OK" = "DE ACUERDO";
+"OK" = "OK";
```

Comment:
also updated this one which I had noticed the other day but didn't open up a PR for